pkg/fission-cli/cmd/package/util/util.go

```diff
@@ -165,12 +165,14 @@ func DownloadURL(fileUrl string) (io.ReadCloser, error) {
 
 func DownloadStrorageURL(ctx context.Context, client cmd.Client, fileUrl string) (io.ReadCloser, error) {
 	var resp *http.Response
-	storagesvcURL, err := util.GetStorageURL(ctx, client)
-	if err != nil {
-		return nil, err
-	}
+	var err error
+
+	if strings.Contains(fileUrl, "/v1/archive?id=") {
```

Comment:
Instead of `Contains` we should parse URL and check path. For better validation
Eg. Code below

```
func validateURL(urlStr string) (bool, error) {
    // Parse the URL string into a URL object
    parsedURL, err := url.Parse(urlStr)
    if err != nil {
        return false, fmt.Errorf("failed to parse URL: %v", err)
    }

    // Check if the path starts with /v1/archive
    if !strings.HasPrefix(parsedURL.Path, "/v1/archive") {
        return false, nil
    }

    // Get query parameters
    queryParams := parsedURL.Query()

    // Check if 'id' parameter exists
    if queryParams.Get("id") == "" {
        return false, nil
    }

    // URL matches all criteria
    return true, nil
}
```